### DISCUSSION TITLE
v2021.09.18

### DISCUSSION THREAD

**Discussion by slawekjaranowski** [s4u/pgp-keys-map#341] [Announcements]
# What's Changed

## New features

- Added keys for jline #326
- Added key for org.codehaus.gmaven #327
- new keys for javassist #328
- new key for org.freemarker #330
- new keys for ognl #329
- new key for Struts 2 #331
- Additional key for org.apache.maven.* #332
- additional key for error_prone_annotations #333
- Additional key for protobuf-java #340

## Dependency updates

- Bump pragmatickm-parent from 1.11.0 to 1.11.1 in /pgp-keys-map-test1 #297
- Bump ao-oss-parent from 1.17.0 to 1.17.1 in /pgp-keys-map-test1 #298
- Bump nmw-oss-parent from 1.14.0 to 1.14.1 in /pgp-keys-map-test1 #299
- Bump semanticcms-parent from 1.13.0 to 1.13.1 in /pgp-keys-map-test1 #300
- Bump spring-boot.version from 2.5.3 to 2.5.4 in /org-springframework-test #301
- Bump mockito-core from 3.11.2 to 3.12.0 in /pgp-keys-map-test1 #302
- Bump stripe-java from 20.69.0 to 20.70.0 in /pgp-keys-map-test1 #303
- Bump libreoffice from 7.1.5 to 7.2.0 in /pgp-keys-map-test1 #305
- Bump mockito-core from 3.12.0 to 3.12.1 in /pgp-keys-map-test1 #306
- Bump jna from 5.8.0 to 5.9.0 in /pgp-keys-map-test1 #308
- Bump mockito-testng from 0.4.8 to 0.4.13 in /pgp-keys-map-test1 #307
- Bump mockito-core from 3.12.1 to 3.12.3 in /pgp-keys-map-test1 #309
- Bump mockito-core from 3.12.3 to 3.12.4 in /pgp-keys-map-test1 #310
- Bump plexus-utils from 3.4.0 to 3.4.1 in /pgp-keys-map-test1 #311
- Bump plexus-utils from 3.4.0 to 3.4.1 in /pgp-keys-map-test1 #312
- Bump stripe-java from 20.70.0 to 20.71.0 in /pgp-keys-map-test1 #313
- Bump apache-jsp from 10.0.7 to 10.0.10 in /pgp-keys-map-test1 #314
- Bump stripe-java from 20.71.0 to 20.72.0 in /pgp-keys-map-test1 #315
- Bump stripe-java from 20.72.0 to 20.73.0 in /pgp-keys-map-test1 #316
- Bump maven-javadoc-plugin from 3.3.0 to 3.3.1 in /org-apache-maven-plugins-test #317
- Bump stripe-java from 20.73.0 to 20.74.0 in /pgp-keys-map-test1 #318
- Bump stripe-java from 20.74.0 to 20.75.0 in /pgp-keys-map-test1 #319
- Bump maven-war-plugin from 3.3.1 to 3.3.2 in /org-apache-maven-plugins-test #320
- Bump ossindex-service-api from 1.7.0 to 1.8.0 in /pgp-keys-map-test1 #322
- Bump tomcat-api from 10.1.0-M4 to 10.1.0-M5 in /pgp-keys-map-test1 #324
- Bump xmlsec from 2.2.2 to 2.2.3 in /pgp-keys-map-test1 #325
- Bump spring-core.version from 5.3.9 to 5.3.10 in /org-springframework-test #334
- Bump ecj from 3.26.0 to 3.27.0 in /pgp-keys-map-test1 #335
- Bump maven-scm-plugin from 1.11.3 to 1.12.0 in /org-apache-maven-plugins-test #337
- Bump libreoffice from 7.2.0 to 7.2.1 in /pgp-keys-map-test1 #339
- Bump stripe-java from 20.75.0 to 20.77.0 in /pgp-keys-map-test1 #338
- Bump tomcat-embed-core from 10.1.0-M4 to 10.1.0-M5 in /pgp-keys-map-test1 #323

# Thanks
Many thanks for collaboration on this release for: @aoindustries,   @slawekjaranowski 

<hr /><em>This discussion was created from the release <a href='https://github.com/s4u/pgp-keys-map/releases/tag/v2021.09.18'>v2021.09.18</a>.</em>

**Comment by ao-apps** [comment]
Thank you for the release.
